README.md

```diff
@@ -658,3 +658,25 @@ It is essential to have a good commit message if you want your change to be revi
 - Add a link to the related jira card (required for any significant automation work)
   - `jira-ticket: https://issues.redhat.com/browse/<jira_id>`
   - The card will be automatically closed once PR is merged
+
+
+## Run basic tests on standard cluster
+To run tests on a standard cluster configuration (more than 1 node is required), use the following command:
+
+```bash
+pytest -m "x86_default_setup" --default-storage-class <cluster default storage class> --skip-artifactory-check
```

Comment:
should we avoid using --skip-artifactory-check in example . this will suppress error

Reply:
the artifactory cannot be used outside rh and without proper creds